doc/user/ruby-managers.md

```diff
@@ -64,16 +64,24 @@ environment variables `GEM_HOME`, `GEM_PATH`, and `GEM_ROOT`. The variables
 are picked up by truffleruby (as any other Ruby implementation would do)
 causing truffleruby to pickup the wrong gem-home directory instead of its own.
 
-It can be easily fixed by clearing the environment with one of the following 
-commands:
+One way to fix this for all sessions is to tell TruffleRuby to ignore `GEM_*`
+variables and always use its own Gem home under `truffleruby/lib/ruby/gems`:
+
+```bash
+# In ~/.bashrc or ~/.zshenv
+export TRUFFLERUBY_RESILIENT_GEM_HOME=true
```

Comment:
I always wondered what this flag was for. Why isn't it called something self-descriptive like `TRUFFLERUBY_IGNORE_GEM_VARIABLES`. I didn't know what resilient was supposed to mean.

Reply:
Right, that would be a clear and better name.
The original idea was "TruffleRuby uses a resilient (able to withstand or recover quickly from difficult conditions) gem home", so even though the environment is completely wrong it will do the right thing.
However, changing the name seems out of scope of this PR and needs changes in quite a few places, so I'd rather do that separately.